src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/ValidationSummaryTagHelper.cs

```diff
@@ -75,9 +76,18 @@ private TagBuilder GetErrorListBuilder()
 
             foreach (var model in orderedStates)
             {
+                var redirectAttribute = ViewContext.ViewData.Model.GetType().GetProperty(model.Key)
+                    ?.GetCustomAttributes<SummaryRedirectAttribute>().FirstOrDefault();
+                
                 foreach (var error in model.Value.Errors)
                 {
-                    var listItem = GetListItemBuilder(model.Key, error.ErrorMessage);
+                    var redirect = model.Key;
```

Comment:
I'm surprised you're not using deconstruction here – I thought you might like that one!